README.md

```diff
@@ -71,3 +71,21 @@ If any of these fail, this will be reported in the GitHub status checks
 interface.
 
 [travis]: https://travis-ci.org/alphagov/govuk-design-system
+
+## Automated Deployment
+We're using Netlify to automate our deployment.
+
+### Production branch
+The Git branch that Netlify uses to build and deploy changes to your site’s main URL, for example govuk-design-system-preview.netlify.com
```

Comment:
Is this talking about the master branch?

Reply:
yep. shall I use "main branch" as a title for this section?

Reply:
Updated, I refer to is as "Master deploy" now, instead of "Production deploy".